src/screens/Inventory/CreateInventoryUpdate.tsx

```diff
@@ -72,18 +75,18 @@ function CreateInventoryUpdate(props: CreateInventoryUpdateProps) {
         </div>
         <form onSubmit={formik.handleSubmit} noValidate>
           <TextField
-            placeholder={'e.g. 5'}
+            placeholder={intl(words.eg_x, "5")}
             required
             type="number"
             unit={product.unit}
-            label={`Updated Amount`}
+            label={intl(words.last_updated_amount)}
```

Comment:
Should be `Updated Amount`. In this case, `last_updated_amount` is less understandable since this is where the user should be putting the new updated amount